src/app/components/gooey-nav/gooeySvg.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default ({
+    id
+}) => {
+  const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
```

Comment:
This is a react anti pattern. You will be better served creating an svg component => 
```
<svg xmlns="http://www.w3.org/2000/svg"  id={id}>
   <defs>
               ....
               ....
     </defs>
</svg>
```